app/assets/javascripts/discourse/app/templates/password-reset.hbs

```diff
@@ -32,13 +32,16 @@
             <div class="alert alert-error">{{this.errorMessage}}</div>
             <br />
           {{/if}}
-          {{#if this.securityKeyRequired}}
+
+          {{#if this.displaySecurityKeyForm}}
             <SecurityKeyForm
               @allowedCredentialIds={{this.model.allowed_credential_ids}}
               @challenge={{this.model.security_key_challenge}}
-              @showSecurityKey={{this.model.security_key_required}}
+              @showSecurityKey={{false}}
```

Comment:
Just pass in a boolean here, we don't want to be mutating `this.model.security_key_required` in this instance but other places that use the component relies on this arg.